test/index.js

```diff
@@ -318,6 +319,24 @@ const HasManyDissociator = require('../src/services/has-many-dissociator');
       });
 
       describe('A simple Line Chart per day on an empty users table', () => {
+        it('should generate a valid SQL query when models mapped to another field', (done) => {
```

Comment:
"when models mapped to another field" should actually be in a `describe` in my opinion:
```
describe('A simple Line Chart per day on an empty users table', () => {
  describe('with a group_by_date_field  field having a conventional name', () => {
    it('should generate a valid SQL query', (done) => {
      // TEST
    });
  });

  describe('with a group_by_date_field  field having an unconventional name', () => {
    it('should generate a valid SQL query', (done) => {
      // TEST
    });
  });
});
```